pages/bellapps.md

```diff
@@ -35,12 +35,13 @@ You will be shown the page below. Make sure you fill it out completely.
 ![Become an Administrator](uploads/images/become_admin.png)
 
 Next, you will need to fill out your configurations. Make sure you write your name all in lowercase letters and your code all in uppercase letters, as in the example below:
```

Comment:
"... as in the example below:" doesn't make as much sense when there is text after it. Maybe you can rework it to be more clear.